README.md

```diff
@@ -5,16 +5,16 @@ Backend for Resource Optimization Service
 
 ## Getting Started
 
-This project uses pipenv to manage the development and production environments.
+This project uses poetry to manage the development and production environments.
 
-Once you have pipenv installed, do the following:
+Once you have poetry installed, do the following:
 
 ```bash
-pipenv install
+poetry install
```

Comment:
@upadhyeammit good work on updating the README.md, I hope we will update the development doc for ROS with same once the PR is merged 🙌🏻